package.json

```diff
@@ -65,5 +74,8 @@
     "ignorePatterns": [
       "lib/**/"
     ]
+  },
+  "dependencies": {
+    "@sanity/pkg-utils": "^1.13.0"
```

Comment:
I don't think it is intentional to add it to both dev and regular dependencies?
```suggestion
```

Reply:
Not intentional – removing 👍